podcasts/End of Year/Stories/Views/CommonStoryViews.swift

```diff
@@ -69,18 +73,37 @@ struct StoryLabel: View {
     private var font: Font {
         switch type {
         case .title:
-            return .system(size: 22, weight: .bold)
+            .custom("DM Sans", size: size).weight(.semibold)
         case .title2:
-            return .system(size: 18, weight: .semibold)
+            .custom("DM Sans", size: size).weight(.semibold)
         case .subtitle:
-            return .system(size: 15, weight: .regular)
+            .custom("DM Sans", size: size).weight(.semibold)
         case .pillarTitle:
-            return .system(size: 14, weight: .bold)
+            .custom("DM Sans", size: size).weight(.bold)
         case .pillarSubtitle:
-            return .system(size: 13, weight: .regular)
+            .custom("DM Sans", size: size).weight(.regular)
         }
     }
 
+    private var size: CGFloat {
+        let iPhone15DefaultHeight: CGFloat = 759
+        let screenHeight = geometry?.size.height ?? iPhone15DefaultHeight
```

Comment:
In case the geometry size is `nil`, it uses the iPhone 15 as a base to calculate the values. I see how that's confusing... but after changing all the stories I can make the  `geometry` parameter not Optional and get rid of that.